wasm/bindings/TranslationRequestBindings.cpp

```diff
@@ -5,7 +5,9 @@
 
 #include <emscripten/bind.h>
 
-#include "TranslationRequest.h"
+#include "response_options.h"
+
+typedef marian::bergamot::ResponseOptions TranslationRequest;
```

Comment:
Bad. Refactor cleanly.

Reply:
I have to draw the line somewhere - if I have to make it clean I have to clean it in extension as well by the rationale. This is the cleanest it in my perspective, so the binding semantics below stay consistent (TranslationRequest -> "TranslationRequest") in JS. The plugin can do their part later? 

My objective here is to get the two different entities "TranslationRequest" | "ResponseOptions" into one of Service so incoming concurrency changes don't wonder why there exist two.

Reply:
Created a separate issue https://github.com/browsermt/bergamot-translator/issues/149 for this.